api/src/main/java/io/strimzi/api/kafka/model/KafkaResources.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+/**
+ * Encapsulates the naming scheme used for the resources which the Cluster Operator manages for a
+ * {@code Kafka} cluster.
+ */
+public class KafkaResources {
+    private KafkaResources() { }
```

Comment:
Why do we need this when all the methods are static?

Reply:
It's pretty normal for a class with only static methods to override the default (compiler-generated) `public` constructor with a `private` one, so that there is no ambiguity in how the class should be used.